float32 deg_turn
---
sensor_msgs/Image camera_image
